Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -28,21 +28,17 @@ import androidx.appcompat.widget.Toolbar
 import androidx.navigation.NavController
 import androidx.navigation.NavDestination
 import androidx.navigation.findNavController
-import androidx.transition.Transition
-import androidx.transition.TransitionListenerAdapter
+import com.google.android.material.transition.Hold
 import com.google.android.material.transition.MaterialSharedAxis
 import com.materialstudies.reply.R
 import com.materialstudies.reply.databinding.ActivityMainBinding
 import com.materialstudies.reply.ui.compose.ComposeFragmentDirections
 import com.materialstudies.reply.ui.email.EmailFragmentArgs
-import com.materialstudies.reply.ui.nav.AlphaSlideAction
-import com.materialstudies.reply.ui.nav.BottomNavDrawerFragment
-import com.materialstudies.reply.ui.nav.ChangeSettingsMenuStateAction
-import com.materialstudies.reply.ui.nav.HalfClockwiseRotateSlideAction
-import com.materialstudies.reply.ui.nav.HalfCounterClockwiseRotateSlideAction
-import com.materialstudies.reply.ui.nav.ShowHideFabStateAction
+import com.materialstudies.reply.ui.nav.*
```

Comment:
Can we keep these as explicit imports?

Reply:
If I remember correctly, explicit imports were a style we were trying to maintain in these samples

Reply:
yep! was having issues with IDE formatting